projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -288,32 +293,27 @@
 .go-table__filter-container {
   display: flex;
   flex-wrap: wrap;
-  padding: 0 1.25rem;
+  padding: 0 $go-table__cell-horizontal-padding;
 }
+
 // Hierarchical Table
-.hierarchical-go-table__body-row:first-child {
-  border-top: 1px solid $theme-light-app-bg;
-}
-.hierarchical-go-table__body-row:last-child {
-  border-bottom: 0;
-}
-.fixed-even-row {
-  background: #ffff;
-}
-.fixed-odd-row {
-  background: rgba($theme-light-app-bg, 0.5);
-}
-.active-parent-row {
-  background-color: #f6f6f6;
-  border-top: 1px solid #f0f0f0;
-}
-.active-row {
-  border-left: 4px solid #65b360;
-  padding: 5.7rem 0rem 0rem 0rem;
```

Comment:
By going the route I am proposing, we wouldn't need most of these. Also, for your information, we use the [BEM CSS naming convention](http://getbem.com/introduction/) so I renamed these classes to follow that convention.

A couple other additional notes for writing CSS within Goponents:
- We always avoid using hard-coded values for colors, you can find all our color variables in `_variables.scss`
- We don't actually use the Tangoe brand color anywhere. When something needs to be branded as the "brand color" then we use the `GoConfigService` and get the `brandColor` for the current configuration